src/main/kotlin/org/jetbrains/research/ml/ast/transformations/PyUtils.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.research.ml.ast.transformations
+
+import com.jetbrains.python.psi.LanguageLevel
+import com.jetbrains.python.psi.PyElementGenerator
+import com.jetbrains.python.psi.PyIfPart
+import com.jetbrains.python.psi.PyIfStatement
+
+object PyUtils {
+    fun createPyIfElsePart(ifElsePart: PyIfPart): PyIfPart {
```

Comment:
It's a matter of taste, but I'd rather use an extension function for `PyIfPart`. Then we can delete the `assert` (or `require`) part. What do you think?

Reply:
In that extension we can't delete assert because `isElif` is a function inside `PyIfPart` interface. I prefer using functions especially if it's a factory function.

Reply:
Ok, let use your version :)